Jenkinsfile

```diff
@@ -4,8 +4,7 @@ library 'pipeline-library'
 def canPublish = env.BRANCH_NAME == '1_X'
 
 buildNPMPackage {
-  labels = 'osx && git && npm-publish && xcode'
-  downstream = ['../appc-cli']
+  labels = 'osx-11 && git && npm-publish'
```

Comment:
Seeing this in use makes me realise osx-11 is completely wrong... but it's in keeping with the existing format at least 🤷‍♂️